onvm/onvm_mgr/onvm_args.c

```diff
@@ -79,6 +79,9 @@ uint8_t global_verbosity_level = 1;
 /* global flag for enabling shared core logic - extern in init.h */
 uint8_t ONVM_NF_SHARE_CORES = 0;
 
+/* global flag for jumbo frames - extern in init.h */
+uint8_t jumbo_frames = 0;
```

Comment:
Please make this `ONVM_USE_JUMBO_FRAMES` to be consistent with our other variables

Reply:
> Please make this `ONVM_USE_JUMBO_FRAMES` to be consistent with our other variables

Okay, I will do so.